entity-management-solr/src/main/java/eu/europeana/entitymanagement/solr/model/SolrTimespan.java

```diff
@@ -0,0 +1,146 @@
+package eu.europeana.entitymanagement.solr.model;
+
+import java.util.List;
+import java.util.Map;
+
+import org.apache.solr.client.solrj.beans.Field;
+
+import eu.europeana.entitymanagement.definitions.model.Timespan;
+import eu.europeana.entitymanagement.vocabulary.AgentSolrFields;
+import eu.europeana.entitymanagement.vocabulary.ConceptSolrFields;
+import eu.europeana.entitymanagement.vocabulary.EntitySolrFields;
+import eu.europeana.entitymanagement.vocabulary.TimespanSolrFields;
+
+public class SolrTimespan extends Timespan {
+	
+	private String payload;
+
+	public SolrTimespan() {
+		super();
+	}
+
+	public SolrTimespan(Timespan timespan) {
+		super();
```

Comment:
use super(timespan) and remove the use of redundant setters

Reply:
Done.